submissions/DmytryjK/task_js_dom/index.html

```diff
@@ -0,0 +1,105 @@
+    <title>JS DOM Kottans</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;500;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+            <div class="wrapper">
+                <a class="header__logo-link" href="/index.html">
+                    <img class="header__logo-img" src="images/LogoPhoto.svg" alt="logo site">
+                </a>
+                <div class="header__hamburger">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main class="main">
+        <div class="container">
+            <div class="wrapper">
+                <nav class="main__navigation">
+                    <ul class="main__navigation-list">
+                        <li class="main__navigation-item">
+                            <button class="main__navigation-button main__navigation__button-active">Бразилія</button>
+                        </li>
+                        <li class="main__navigation-item">
+                            <button class="main__navigation-button">Гонконг</button>
+                        </li>
+                        <li class="main__navigation-item">
+                            <button class="main__navigation-button">Китай</button>
+                        </li>
+                        <li class="main__navigation-item">
+                            <button class="main__navigation-button">США</button>
+                        </li>
+                        <li class="main__navigation-item">
+                            <button class="main__navigation-button">Канада</button>
+                        </li>
+                        <li class="main__navigation-item">
+                            <button class="main__navigation-button">Франція</button>
+                        </li>
+                        <li class="main__navigation-item">
+                            <button class="main__navigation-button">Гренландія</button>
+                        </li>
+                        <li class="main__navigation-item">
+                            <button class="main__navigation-button">Норвегія</button>
+                        </li>
+                    </ul>
+                </nav>
+                <div class="main__content">
+                    <div class="main__content-block"></div>
+                    <div class="main__content-block"></div>
+                    <div class="main__content-block"></div>
+                    <div class="main__content-block"></div>
+                    <div class="main__content-block"></div>
+                    <div class="main__content-block"></div>
```

Comment:
If your albom consist of thousands photos that must be thousands empty elements. Try create one element in which you can set to view some photo information.

Reply:
Above, I just described the reason why I added empty blocks. I don’t know how to get around this crutch so that the animation works.